__tests__/app.test.js

```diff
@@ -2,6 +2,12 @@ import pool from '../lib/utils/pool.js';
 import setup from '../data/setup.js';
 import request from 'supertest';
 import app from '../lib/app.js';
+import Pokemon from '../lib/models/HiddenAbility.js';
+import Holiday from '../lib/models/Holiday.js';
+import Book from '../lib/models/Book';
+import Cat from '../lib/models/Cat';
+import VideoGame from '../lib/models/VideoGame';
+
 
 describe('pokemon routes', () => {
```

Comment:
I would probably suggest splitting this up into 5 different test files, just to do the single responsibility principle, so that each test tests one model.